src/CalcViewModel/UnitConverterViewModel.cpp

```diff
@@ -515,8 +515,10 @@ void UnitConverterViewModel::OnPasteCommand(Platform::Object ^ parameter)
     // Ensure that the paste happens on the UI thread
     // EventWriteClipboardPaste_Start();
     // Any converter ViewMode is fine here.
-    CopyPasteManager::GetStringToPaste(m_Mode, NavCategory::GetGroupType(m_Mode))
-        .then([this](String ^ pastedString) { OnPaste(pastedString); }, concurrency::task_continuation_context::use_current());
+    
+    auto that(this);
```

Comment:
Instead can you lambda capture this by value? c++17 feature, I'm not sure if it works with CX.

[*this](String ^ pastedString) { this->... }

Reply:
Same feedback for UnitConverter.xaml.cpp:246

Reply:
I tried but it's not possible :(
`error C3076: you cannot embed an instance of a reference type`, same with `[&]` and `[=]`.

`that` is the only workaround I know at the moment.